src/nodes.coffee

```diff
@@ -1451,9 +1449,7 @@ exports.Param = class Param extends Base
   # `name` is the name of the parameter and `node` is the AST node corresponding
   # to that name.
   eachName: (iterator, name = @name)->
-    atParam = (obj) ->
-      node = obj.properties[0].name
-      iterator node.value, node unless node.value.reserved
+    atParam = (obj) -> iterator "@#{obj.properties[0].name.value}", obj
```

Comment:
Can't we just no-op here? Would a consumer of this function expect to get `"@name"` for an at-param or just have it omitted?